#goal definition
std_msgs/String folder_path
std_msgs/String folder_pano_path
std_msgs/String direction

---
#result definition
bool finished
---
#feedback
#int32[] sequence
